### PR TITLE
updated about section

### LINKED ISSUE #7: Update 'About' section
- Update copy from the [2021 Website Copy](https://docs.google.com/document/d/1S1W0nse79HzekyUZJOAXUwJuBVSMPoqjbTCMaEkaBBs/edit?usp=sharing) doc
- About - Reuse + restyle `Section.tsx`
- Stats - Reuse + restyle `Stats.tsx`
- Code the Future - might need to extend Section to support images that touch the edges (currently, images exist within the Bootstrap container only) to ensure that the rocket lines up with the blue trail

### PR DESCRIPTION
**Pull request by gracewgao**
closes #7 

- still more to be done + discussed regarding this section but here are the changes for now
- to-do:
     - should we break up the `Section` component into customized component variations? it's a good layout template but it might be easier to make things align if we have more customization freedom
     - assets from Ananya + copy will be updated later
     - the highlight colour in the dark theme should probably be changed from the purple we have currently - I'm thinking yellow or light teal might work better - any thoughts?

### CODE REVIEW

**Comment by gracewgao** [comment]
so this PR actually turned into a very large dump of new assets + styling

**Comment by gracewgao** [comment]
there are a bunch of unused assets to clean up and probably some optimization with the code but it gets the job done lol

**Comment by gracewgao** [comment]
I did move some assets around so lmk what you think about the placement of the new graphics

File: sections/About2.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { Col, Container, Row, Image } from "react-bootstrap";
+import { ButtonProps, Heading, Text, Link, Button } from "../components";
+import styled from "styled-components";
+import media from "../theme/media";
+
+type About1Props = {
```
Comment (malav-mehta) [inline sections/About2.tsx:6]: Probably rename `About1` to `About2` throughout the file, even though it makes no difference

**Review by malav-mehta — APPROVED** [review]

**Pull request merged**